### PR TITLE
fix: TLS_ENABLED might not be defined

### PR DESCRIPTION
**Pull request by leon-inf**
resolve https://github.com/apecloud/kubeblocks/issues/8598

### CODE REVIEW

**Review by sophon-zt — DISMISSED** [review]

**Review by wangyelei — DISMISSED** [review]

File: addons/clickhouse/configs/ch_keeper_00_default_overrides.xml.tpl [hunk 1]
```diff
@@ -30,14 +30,14 @@
           <raft_logs_level>warning</raft_logs_level>
       </coordination_settings>
       <raft_configuration>
-      {{- if eq $.TLS_ENABLED "true" }}
+      {{- if and (index . "TLS_ENABLED") (eq .TLS_ENABLED "true") }}
```
Comment (wangyelei) [inline addons/clickhouse/configs/ch_keeper_00_default_overrides.xml.tpl:33]: render failed with nest scope, define a var to resolve it

**Review by zjx20 — APPROVED** [review]

File: addons/clickhouse/configs/ch_keeper_00_default_overrides.xml.tpl (hunk 1 above)
Comment (leon-inf) [reply addons/clickhouse/configs/ch_keeper_00_default_overrides.xml.tpl:33]: Fixed

**Pull request merged**